hack/test-e2e.sh

```diff
@@ -32,21 +38,30 @@ https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prom
 oc create -n ${NAMESPACE} -f \
 https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml || :
 
-repo_dir="$(dirname $0)/.."
-$repo_dir/hack/gen-olm-artifacts.sh ${CSV_FILE} ${NAMESPACE} 'ns' | oc create -f - || :
+repo_dir=$(echo "$(cd "$(dirname "$1")"; pwd -P)")
+oc create ns ${NAMESPACE} || :
 
-$repo_dir/hack/gen-olm-artifacts.sh ${CSV_FILE} ${NAMESPACE} >> ${manifest}
-sed -i "s,quay.io/openshift/origin-cluster-logging-operator:latest,${IMAGE_CLUSTER_LOGGING_OPERATOR}," ${manifest}
-if [ -n "${IMAGE_FORMAT:-}" ] ; then
-  for comp in logging-curator5 logging-elasticsearch5 logging-fluentd logging-kibana5 logging-oauth-proxy logging-rsyslog ; do
-    img=$(sed -e "s,\${component},$comp," <(echo $IMAGE_FORMAT))
-    sed -i "s,quay.io/openshift/origin-${comp}:latest,${img}," ${manifest}
-  done
-fi
+deploy_operator(){
+  local index=$1
+  local manifest=$2
+  local createOperatorGroup=$3
+  local image=$4
+  echo "Deploying operator from ${manifest}"
+  CONFIGMAP_NAME=openshift-olm-test-${index} \
+  SUFFIX="-${index}" \
+  VERSION=${VERSION} \
+  OPERATOR_IMAGE=${image:-} \
+  MANIFEST_DIR=${manifest} TEST_NAMESPACE=${NAMESPACE} TARGET_NAMESPACE=${NAMESPACE} ${repo_dir}/hack/vendor/olm-test-script/e2e-olm.sh
+  if [ "$?" != "0" ] ; then
+    echo "Error deploying operator via OLM using manifest: $manifest"
+    exit 1
+  fi
+}
+
+deploy_operator 0 "${repo_dir}/manifests" "true" "${IMAGE_CLUSTER_LOGGING_OPERATOR}"
 
-$repo_dir/hack/gen-olm-artifacts.sh ${CSV_FILE} ${NAMESPACE} 'crd' >> ${global_manifest}
-$repo_dir/hack/gen-olm-artifacts.sh ${EO_CSV_FILE} ${NAMESPACE} 'crd' >> ${global_manifest}
-$repo_dir/hack/gen-olm-artifacts.sh ${EO_CSV_FILE} ${NAMESPACE} >> ${manifest}
+oc -n openshift-cluster-version scale deployment cluster-version-operator --replicas=0
+oc -n openshift-operator-lifecycle-manager scale deployment olm-operator --replicas=0
```

Comment:
You probably need some sort of `wait` here - e.g. https://github.com/openshift/origin-aggregated-logging/blob/master/hack/deploy-logging.sh#L208

Reply:
I think i may be able to revert this.  This was added to handle the upgrade portion of the test but I think we should fix the upgrade part by going through a typical OLM workflow which would make this unnecessary.